region/maridia/outer/Mama Turtle Room.json

```diff
@@ -886,6 +903,52 @@
       ],
       "flashSuitChecked": true,
       "note": "To save energy, start from the upper right ledge; it may require repositioning Mama Turtle to the right to get to the ledge."
+    },
+    {
+      "link": [5, 6],
+      "name": "Base",
+      "requires": [
+        "canUseGrapple",
+        "SpaceJump"
+      ]
+    },
```

Comment:
the left->right version of this is easy because the grapple block is an easy target from the left